packages/fuselage/src/components/SidebarV2/SidebarLink.tsx

```diff
@@ -18,6 +18,8 @@ export const SidebarLink = ({
   menu?: ReactNode;
 } & LinkHTMLAttributes<HTMLAnchorElement>) => (
   <a
+    role='link'
+    tabIndex={0}
```

Comment:
are these attributes needed here? the element is already `<a />` - which has role link and tabIndex 0 by default, isn't it?